Yank/tests/test_restraints.py

```diff
@@ -183,7 +183,7 @@ def test_harmonic_standard_state():
     LJ_fluid = testsystems.LennardJonesFluid()
 
     # Create Harmonic restraint.
-    restraint = yank.restraints.create_restraint('Harmonic', restrained_receptor_atoms=[1])
+    restraint = yank.restraints.create_restraint('Harmonic', restrained_receptor_atoms=1)
```

Comment:
Is there a test somewhere where a list of int (even if its len == 1) is checked? If not, it would be good to make one to ensure functionality.

Reply:
Yep! There's one in `test_restraints.test_partial_parametrization`.